examples/vpc-with-multiple-az/monitoring/README.md

```diff
@@ -0,0 +1,15 @@
+# GraphDB AWS Monitoring Module
+
+This module creates the necessary configurations to set up a Cloudwatch dashboard with all the necessary widgets that show the health of the graphdb cluster. Those configurations should be by tenant.
```

Comment:
The description doesn't match the dashboard JSON

Reply:
This is supposed to happen in the next iterations ;) but can remove it for now